src/ripple/ledger/impl/CashDiff.cpp

```diff
@@ -330,6 +330,16 @@ class CashDiff::Impl
             || rhsDiffs_.hasDiff();
     }
 
+    // For tiny offers of TakerPays IOU and TakerGets XRP, cases have been
+    // observed where XRP rounding allows a tiny amount of IOU to be
+    // removed from an Offer while returning no XRP to the offer owner.
+    // That's because the XRP amount was rounded down to zero drops.
+    //
+    // The person submitting the tiny offer does not, however, get something
+    // for nothing.  The transaction's fee is significantly larger than the
+    // value of the received IOU.
+    int xrpRoundToZero () const;
```

Comment:
This comment would be equally valuable in CashDiff.h.

Reply:
@ximinez , would you like to see the comment moved into CashDiff?  Or should there be a copy of the comment in CashDiff?  I'm fine with either or with leaving it alone...

Reply:
I think moving it would make sense.